tools/depends/configure.ac

```diff
@@ -357,14 +357,8 @@ case $host in
 
         # setup which sdk to use
         if test "$use_platform" = "tvos"; then
-          # setup which cpu to use
-          if test "x$use_cpu" = "xauto"; then
-            use_cpu=arm64
-          fi
-
+          use_cpu=arm64
```

Comment:
Why remove the auto cpu selection?

Reply:
Currently we only have the one cpu for tvos. If apple changes, or someone introduces tvos simulator support (x64), why not just set to what we actually can use